### PR TITLE
Fixes for emoji remix

### PR DESCRIPTION
**Pull request by bacontech**
Style emjoi remix

Also: Fix history hover/tooltip

(carrot thing is off center - but better anyway)

image

image

image

### CODE REVIEW

File: src/app/components/game-board/game-board.component.html [hunk 1]
```diff
@@ -14,21 +14,12 @@
...
           <span *ngSwitchCase="GameType.WORDS">{{ tile.word }}</span>
-          <span *ngSwitchCase="GameType.MEMES">
-            <ion-img src="./assets/memes/{{tile.word}}.jpg"></ion-img>
-          </span>
-          <span *ngSwitchCase="GameType.PICTURES">
-            <ion-img src="./assets/pictures/smaller/{{tile.word}}.png"></ion-img>
-          </span>
-          <span *ngSwitchCase="GameType.EMOJI_REMIX">
-            <ion-img src="./assets/emoji-remix/{{tile.word}}.png"></ion-img>
-          </span>
```
Comment (jjzabkar) [inline src/app/components/game-board/game-board.component.html:25]: Won't this break Emojis?

Comment (bacontech) [reply src/app/components/game-board/game-board.component.html:25]: No :)

**Comment by bacontech** [comment]
@jjzabkar 
Picture of game with regular emojis

image

**Review by jjzabkar — APPROVED** [review]
🚢

**Review by koldsco — APPROVED** [review]

**Pull request merged**